saleor/graphql/views.py

```diff
@@ -39,11 +37,26 @@
 
 INT_ERROR_MSG = "Int cannot represent non 32-bit signed integer value"
 
+meter.create_metric(
+    "saleor.graphql_queries",
+    service_scope=True,
+    type=MetricType.COUNTER,
+    unit=Unit.REQUEST,
+    description="Number of GraphQL queries.",
+)
+meter.create_metric(
+    "saleor.graphql_query_duration",
+    service_scope=True,
+    type=MetricType.HISTOGRAM,
+    unit=Unit.MILLISECOND,
+    description="Duration of GraphQL queries.",
+)
+
```

Comment:
Maybe we should define metrics in separate files? e.g `saleor/graphql/metrics.py` ?